go/pserver/optimizer.go

```diff
@@ -42,12 +42,12 @@ func newOptimizer(paramWithConfigs ParameterWithConfig) *optimizer {
 	c := paramWithConfigs.Config
 	log.WithFields(log.Fields{
 		"ElementType": p.ElementType,
-		"ParamSize":   len(p.Content),
+		"ParamSize":   len(p.Content) / C.sizeof_float,
 		"ConfigSize":  len(c),
 	}).Info("New Optimizer Created with config:")
 	var cbuffer unsafe.Pointer
 	cbuffer = C.malloc(C.size_t(len(p.Content)))
-	C.memcpy(cbuffer, unsafe.Pointer(&p.Content[0]), C.size_t(len(p.Content)))
+	C.memcpy(cbuffer, unsafe.Pointer(&p.Content[0]), C.size_t(len(p.Content)/C.sizeof_float))
```

Comment:
Oh, no,  we malloc four times memory.......
`cbuffer = C.malloc(C.size_t(len(p.Content)))`, it absolutly will run smoothly because I waste the rest buffer.
`weird`, by the way. : )